### PR TITLE
test(client): port blog update

### PR DESCRIPTION
**Pull request by danstarns**
1. Ports the 'blog-update' test to the new functional style
2. Adds fakerjs for random field content
3. Covers more operations like; simple updates, connect and disconnect. 
4. Removes snapshots and uses `expect.equal`

Closes: https://github.com/prisma/client-planning/issues/39

### CODE REVIEW

File: packages/client/tests/functional/blog-update/tests.ts [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+      },
+    })
+
+    expect(response).toMatchObject({
+      ...user,
+      posts: [],
+    })
+  })
+
+  test('should create a user with posts and a profile and update itself and nested connections setting fields to null', async () => {
```
Comment (danstarns) [inline packages/client/tests/functional/blog-update/tests.ts:167]: This is the original test

File: packages/client/tests/functional/blog-update/_matrix.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { defineMatrix } from '../_utils/defineMatrix'
+
+export default defineMatrix(() => [
```
Comment (danstarns) [inline packages/client/tests/functional/blog-update/_matrix.ts:3]: Wating for https://github.com/prisma/prisma/pull/13507 to cover the rest

**Review by SevInf — APPROVED** [review]
LGTM, with some minor suggestions

File: packages/client/tests/functional/_utils/setupTestSuiteMatrix.ts [hunk 3]
```diff
@@ -6,6 +6,10 @@ import { MatrixOptions } from './types'
 
 export type TestSuiteMeta = ReturnType<typeof getTestSuiteMeta>
 
+export type MatrixOptions = {
+  skipDb?: boolean
```
[the remark this answers is not in the document]
Comment (danstarns) [reply packages/client/tests/functional/_utils/setupTestSuiteMatrix.ts]: Good catch!

**Pull request merged**